sources/Filer/DropZoneTab.h

```diff
@@ -9,29 +9,49 @@
 #ifndef DROPZONETAB_H
 #define DROPZONETAB_H
 
+#include <Button.h>
+#include <StringView.h>
 #include <View.h>
 
 const pattern stripePattern = {0xcc, 0x66, 0x33, 0x99, 0xcc, 0x66, 0x33, 0x99};
 
+class _EXPORT DropZone;
+
```

Comment:
I removed that completely. Still works; no idea why I added that in the first place... I might have read it in some other code.